tests/foreman/api/test_user.py

```diff
@@ -922,10 +921,9 @@ def test_positive_access_entities_from_ipa_org_admin(self):
         user.update(['role'])
         with self.assertNotRaises(HTTPError):
             for entity in [
-                entities.Architecture, entities.Audit, entities.Bookmark,
-                entities.CommonParameter, entities.DockerComputeResource,
-                entities.LibvirtComputeResource, entities.OVirtComputeResource,
-                entities.VMWareComputeResource, entities.ConfigGroup, entities.DockerHubContainer,
-                entities.DockerRegistryContainer, entities.Errata, entities.OperatingSystem
+                entities.Architecture, entities.Audit, entities.Bookmark, entities.CommonParameter,
+                entities.DockerComputeResource, entities.LibvirtComputeResource,
+                entities.OVirtComputeResource, entities.VMWareComputeResource,
+                entities.ConfigGroup, entities.Errata, entities.OperatingSystem
```

Comment:
I know this is not the point of your PR, but can you move that list of entities into fixture or class variable? It's identical for both tests, so it's better to share it instead of copy-pasting it.
(and I mean both tests in `tests/foreman/api/test_user.py`; I'm ok with repeating it here and in `tests/foreman/api/test_role.py`)

Reply:
- it's in 2 distinct classes FreeIPAUserTestCase and ActiveDirectoryUserTestCase -> no class var possible
- old unittest style -> no fixture possible

Can be put in constants, can be ....